source-map.bs

```diff
@@ -433,21 +427,35 @@ However, It is unclear what a "source map reference" looks like in anything othe
 More specifically, what a source map reference looks like in a language that doesn't support
 JavaScript-style single-line comments.
 
-JSON over HTTP Transport
-========================
-
-For historic reasons, when delivering source maps over HTTP, servers may prepend a line
-starting with the string `)]}'` to the source map.  A client fetching a source map via
-HTTP thus should check if the response starts with this string and then ignore the
-first line.
-
-```
-)]}'garbage here
-{"version": 3, ...}
-```
-
-Is to be interpreted as
-
-```
-{"version": 3, ...}
-```
+Fetching Source Maps {#fetching-source-maps}
+============================================
+
+To fetch a source map given a [=URL=] |url|, run the following steps:
+
+1. Let |promise| be [=a new promise=].
+1. Let |request| be a new [=request=] whose [=request/URL=] is |url|.
+1. [=Fetch=] |request| with [=processResponseConsumeBody=] set to the following steps given [=response=] <var ignore>response</var> and null, failure, or a [=byte sequence=] |bodyBytes|:
+    1. If |bodyBytes| is null or failure, [=reject=] |promise| with a {{TypeError}} and abort these steps.
+    1. If |url|'s [=url/scheme=] is "<code>http</code>" and |bodyBytes| [=byte sequence/starts with=] \`<code>)]}'</code>\`, then:
+        1. [=While=] |bodyBytes|'s [=byte sequence/length=] is not 0 and |bodyBytes|'s 0th byte is not 0x0A (LF) or 0x0D (CR):
+            1. remove the 0th byte from |bodyBytes|.
```

Comment:
- Do we know if this is only for HTTP or also for HTTPS?
- Do we know what "ignore the first line" means? Some possible sets of line terminators are:
  - `\n` and `\r` ("common" terminators)
  - `\n`, `\r`, U+2028, U+2029 (what JS uses as line terminators)

Reply:
Does anyone have bandwidth to test this in some browsers?

Reply:
According to https://developer.chrome.com/blog/sourcemaps#potential_xssi_issues, the answer could also be "just \n"